app/factories/user.py

```diff
@@ -0,0 +1,27 @@
+import factory
+from app.models.user import db, User
+import app.factories.common as common
+
+
+class UserFactory(factory.alchemy.SQLAlchemyModelFactory):
+    class Meta:
+        model = User
+        sqlalchemy_session = db.session
+
+    email = common.email_
```

Comment:
Yes, we can. Factory boy also has support for all things in the Faker library. There's no point adding it for the dredd testing of api docs, as there is only going to be one resource. We can use the sequence and faker also when writing unit tests for validation and the permission decorators